communication/moderators.md

```diff
@@ -197,12 +198,13 @@ Moderators pro tempore seats: 10
 
 Administrators seats: 4
```

Comment:
this needs an update too. how many admins do we need?

Reply:
We can set that to whatever we want, the initial number was just an educated guess.